content/articles/using-multiple-ssh-keys-for-multiple-github-accounts/index.md

```diff
@@ -0,0 +1,129 @@
+
+```ssh
+id_rsa_personal
+id_rsa_work
+```
+
+After generating the keys, use the following command to check if all keys were created.
+
+`ls ~/.ssh`
+
+The following files list is presented.
+
+![Generating a files list](/engineering-education/using-multiple-ssh-keys-for-multiple-github-accounts/ls-ssh.jpg)
+
+#### Adding a new SSH key to your GitHub account
+Now that we have the SSH keys, let us link them with the Github account.
+To obtain the SSH key execute this command:
+
+`cat id_rsa_personal.pub`
+
+![Getting SSH key](/engineering-education/using-multiple-ssh-keys-for-multiple-github-accounts/ssh-copy-key.jpg)
+
+Copy the SSH key and then sign in to your GitHub account. Follow the steps below to add an SSH key to your GitHub account.
+1. On your GitHub, navigate to Settings
+2. Choose SSH and GPG keys
+3. Hit on button New SSH Key, give a significant title, and paste the key
+4. Finally, click the Add Key button
+
+![Adding SSH key to your Github account](/engineering-education/using-multiple-ssh-keys-for-multiple-github-accounts/add-sshkey.jpg)
+
+### Creating and updating the SSH config file
+Finally, let us bring it all together in a config file. There are two GitHub accounts, the personal and work accounts. The personal account is the local account, and work is the global account.
+
+The SSH config file is accessed by running this command:
+
+`~/.ssh/config`
+
+If it exists, you can edit it, or else it can be created using this command:
+
+```ssh
+touch config
+nano config
+```
+
+Update the config file by adding the following rules:
+
+```ssh
+# Personal account, - the default config
+Host github.com-personal github account
+   HostName github.com
+   User git
+   IdentityFile ~/.ssh/id_rsa_personal
+   
+# Work account
+Host github.com-organization github account   
+   HostName github.com
+   User git
+   IdentityFile ~/.ssh/id_rsa_work
+```
+
+![Adding a config file](/engineering-education/using-multiple-ssh-keys-for-multiple-github-accounts/config.jpg)
+
+In the code above, we have two different values. One is the work repository, and another for the user's repository. The values enable you to add and update the SSH config of the GitHub repositories.
+
+### Cloning the Repositories
+Finally, we have to clone repositories. While cloning, make sure that you use the hostnames that we used during SSH configuration. To clone the repositories, execute the below clone command.
+
+#### Cloning personal repository
+Cloning your personal private projects, you use this command.
+`git clone git@github.com:your-github-account/private-project-repo.git`
+
+#### Cloning work repository
+Cloning your company's projects, you will use this command.
+`git clone git@github.myorganization.com:org-account/company-project-repo.git`
+
+### Conclusion
+That's all there is for this article. I hope it will be helpful. Also, I hope it helps you understand how to use multiple SSH keys for multiple GitHub accounts.
```

Comment:
Give a brief summary of what you mentioned previously. Write key takeaways from this article. Tell how and where the reader could learn more about this.

If needed, you can add an additional `Further reading` section.